hatchways/team-fuji
# Feature amazon s3 #25

### What this PR does:
- Enables users to upload a profile Image

### Screenshots / Videos:
![Screen Shot 2021-06-07 at 12 23 47 AM](https://user-images.githubusercontent.com/58789967/120958927-aec89e00-c726-11eb-8e5b-35f1a0feca86.png)
![Screen Shot 2021-06-07 at 12 23 57 AM](https://user-images.githubusercontent.com/58789967/120958930-b0926180-c726-11eb-833b-3f993aa78661.png)
![Screen Shot 2021-06-07 at 12 13 38 AM](https://user-images.githubusercontent.com/58789967/120958272-49c07880-c725-11eb-8402-96a046d30a9a.png)
![Screen Shot 2021-06-07 at 12 15 07 AM](https://user-images.githubusercontent.com/58789967/120958311-6e1c5500-c725-11eb-8363-54df7416537f.png)

### Any information needed to test this feature:
  1. Login/Create an account
  2. Head to the options menu next to the profile photo, select profile Image
  3. Drag or select a profile Image, notice that dropping anything other than an Image will produce an error
  4. Hit submit and Watch app state update automatically with new Profile Photo
